Assignment5/CarAccidentErrorHandling.java

```diff
@@ -0,0 +1,52 @@
+public class CarAccidentErrorHandling {
+
+    public static void main(String[] args) {
+        try {
+            simulateCarAccident(true);
```

Comment:
Can we have single try catch instead of multiple try catches because then there are multiple finally which will be excecuted whether or not exception occurs or not.  This way we will have single finally block.